modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2TokenRevocationAfterAccountDisablingTestCase.java

```diff
@@ -0,0 +1,444 @@
+
+    private final String OAUTH_APPLICATION_NAME_1 = "oauthTestApplication1";
+    private final String OAUTH_APPLICATION_NAME_2 = "oauthTestApplication2";
+    private final String APP_CALLBACK_URL = "http://localhost:8490/playground2/oauth2client";
+    private final String OAUTH_VERSION_2 = "OAuth-2.0";
+
+    private static final String SERVICE_PROVIDER_1_NAME = "PlaygroundServiceProvider1";
+    private static final String SERVICE_PROVIDER_2_NAME = "PlaygroundServiceProvider2";
+    private static final String TEST_NONCE = "test_nonce";
+
+    private CloseableHttpClient client;
+
+    @DataProvider
+    public static Object[][] oAuthConsumerApplicationProvider() {
+
+        return new Object[][] {
+                {TestUserMode.SUPER_TENANT_ADMIN},
+                {TestUserMode.TENANT_ADMIN}
+        };
+    }
+
+    @Factory(dataProvider = "oAuthConsumerApplicationProvider")
+    public OAuth2TokenRevocationAfterAccountDisablingTestCase(TestUserMode userMode)
+            throws Exception {
+
+        super.init(userMode);
+        AutomationContext context = new AutomationContext("IDENTITY", userMode);
+        this.adminUsername = context.getContextTenant().getTenantAdmin().getUserName();
+        this.adminPassword = context.getContextTenant().getTenantAdmin().getPassword();
+        this.activeTenant = context.getContextTenant().getDomain();
+        this.tokenType = "Default";
+    }
+
+    @BeforeClass(alwaysRun = true)
+    public void testInit() throws Exception {
+
+        createServiceProviderApplication();
+        addNewTestUserWithRole();
+        ConfigurationContext configContext = ConfigurationContextFactory
+                .createConfigurationContextFromFileSystem(null, null);
+        applicationManagementServiceClient =
+                new ApplicationManagementServiceClient(sessionCookie, backendURL, configContext);
+        client = HttpClientBuilder.create().disableRedirectHandling().build();
+        idPMgtClient = new IdentityProviderMgtServiceClient(sessionCookie, backendURL);
+        residentIDP = idPMgtClient.getResidentIdP();
+        usmClient = new RemoteUserStoreManagerServiceClient(backendURL, sessionCookie);
+        userIdentityManagementAdminServiceClient = new UserIdentityManagementAdminServiceClient(backendURL,
+                sessionCookie);
+        AuthenticatorClient logManager = new AuthenticatorClient(backendURL);
+        String secondaryTenantDomain = isServer.getTenantList().get(1);
+        String tenantCookie = logManager.login(ADMIN + "@" + secondaryTenantDomain,
+                ADMIN, isServer.getInstance().getHosts().get("default"));
+        tenantIDPMgtClient = new IdentityProviderMgtServiceClient(tenantCookie, backendURL);
+    }
+
+    @Test(description = "Create access tokens")
+    public void testCreateAccessTokens() throws Exception {
+
+        Set<String> appKeys = applications.keySet();
+        for (String app : appKeys) {
```

Comment:
app->appName

Reply:
Fixed in https://github.com/wso2/product-is/pull/14727/commits/4d3f5370d709c1e5df86fce3f4082ba2683b31d5